app/src/main/java/org/supla/android/BaseActivity.java

```diff
@@ -54,7 +56,7 @@ of the License, or (at your option) any later version.
 import io.reactivex.rxjava3.schedulers.Schedulers;
 
 @SuppressLint("registered")
-public class BaseActivity extends Activity implements SuplaClientMessageHandler.OnSuplaClientMessageListener {
+public class BaseActivity extends FragmentActivity implements SuplaClientMessageHandler.OnSuplaClientMessageListener {
```

Comment:
Are you sure we can change Activity to FragmentActivity?

Reply:
Hilt expects FragmentActivity or AppCompatActivity. As we use fragments, I've selected FragmentActivity. With AppCompatActivity there is something wrong with toolbar.